content/intro-to-storybook/react/fr/get-started.md

```diff
@@ -42,7 +42,7 @@ yarn start
 ```
 
 <div class="aside"> 
-Vous avez peut-être remarqué que nous avons ajouté le drapeau <code>--watchAll</code> à notre commande de test, ne vous inquiétez pas, c'est intentionnel, ce petit changement permettra de s'assurer que tous les tests sont effectués et que tout va bien avec notre application. Pendant que vous progressez dans ce tutoriel, vous serez exposé à différents scénarios de test. Pour vous assurer que votre suite de tests fonctionneront dans leur intégralité vous pouvez aussi ajouter ce drapeau à votre script de test dans votre <code>package.json</code> .
+Vous avez peut-être remarqué que nous avons ajouté le drapeau <code>--watchAll</code> à notre commande de test, ne vous inquiétez pas, c'est intentionnel, ce petit changement permettra de s'assurer que tous les tests sont effectués et que tout va bien avec notre application. Pendant que vous progressez dans ce tutoriel, vous serez exposés à différents scénarios de test. Pour vous assurer que votre suite de tests fonctionnera dans son intégralité, vous pouvez aussi ajouter ce drapeau à votre script de test dans votre <code>package.json</code> .
```

Comment:
I think "vous serez exposé" is correct. For the rest good job

Reply:
I'm sure that it's : vous serez exposés.

Reply:
Ok thanks. Good job. Good to be merged

Reply:
Thank you